tests/pytype_blacklist.txt

```diff
@@ -0,0 +1,59 @@
+# Pytype blacklist. Files will not be tested with pytype.
```

Comment:
@matthiaskramm Why are so many files blacklisted? Is there any incentive to reduce this list?